source/dub/dub.d

```diff
@@ -245,13 +233,13 @@ class Dub {
 		scope (exit) environment.remove("DUB_REGISTRY");
 		auto dub = new Dub(".", null, SkipPackageSuppliers.none);
 
-		dub.m_config = new DubConfig(Json(["skipRegistry": Json("none")]), null);
+		dub.m_config.skipRegistry = typeof(dub.m_config.skipRegistry)(SkipPackageSuppliers.none);
```

Comment:
does SetInfo not override opAssign?

Reply:
Only the constructor is overriden. That's on purpose, because people usually don't change those values after they are read.